spec.html

```diff
@@ -38929,7 +38929,7 @@ <h1>synchronizes-with</h1>
       <p>For a candidate execution _execution_, _execution_.[[SynchronizesWith]] is the least Relation on events that satisfies the following.</p>
       <ul>
         <li>
-          For each pair (_R_, _W_) in _execution_.[[ReadsFrom]], (_W_, _R_) is in _execution_.[[SynchronizesWith]] if all the following are true.
+          For each pair (_R_, _W_) in _execution_.[[ReadsFrom]], if (_W_, _R_) is in _execution_.[[SynchronizesWith]], then all the following are true.
```

Comment:
Because synchronizes-with is defined as the "least relation" satisfying the conditions, the previous formulation was more accurate, even taking the material implication interpretation of "if". The least relation satisfying the modified first bullet would be the entirely empty relation.

In my opinion the simplest thing here is to retain the old wording, since the phrasing "least relation" interacts with the other bullets too.

Reply:
Indeed, I had forgotten I put the least relation language for a previous formalist. :)